gensim/scripts/segment_wiki.py

```diff
@@ -278,9 +278,6 @@ def __init__(self, fileobj, min_article_character=200, processes=None,
             Minimal number of character for article (except titles and leading gaps).
         processes : int, optional
             Number of processes, max(1, multiprocessing.cpu_count() - 1) if None.
-        lemmatize : bool, optional
```

Comment:
Should we keep some sort of note? This change will break people's existing code, so a helpful message would be… helpful :)

Reply:
Yes, a no-longer-supported failure message if `lemmatize=True` - perhaps pointing to external lemmatization options – would seem polite.

Reply:
This would go in the change log, right?